local/public/wp-config.php

```diff
@@ -20,7 +20,7 @@
 
 // Multisite config.
 define( 'MULTISITE', true );
-define( 'SUBDOMAIN_INSTALL', false ); // Matching VIP setup.
+define( 'SUBDOMAIN_INSTALL', true );
```

Comment:
@nandotess @bartoszgadomski we made this change on Ziff, but seems like it should be done here as well since you use `network.local.wpenv.net` and `main.local.wpenv.net` in the template setup. Am I understanding that correctly? Let me know your thoughts (cc @landbryo)

Reply:
Sounds like you're right

Reply:
I would prefer to keep this mathing VIP.

[setup-multisite.sh](https://github.com/xwp/vip-site-template/blob/main/local/scripts/setup-multisite.sh#L17-L26) changes this to `true` during setup and revert it back at the [end](https://github.com/xwp/vip-site-template/blob/main/local/scripts/setup-multisite.sh#L76-L86) of the script.

I need to be honest I don't know the issues having this set to `false`. But I know for sure this should not block development and site creation. On Vox, this is still `false`. On DK this is `false` and localhost setup is working.